manifests/webhook.yaml

```diff
@@ -6,12 +6,21 @@ metadata:
   labels:
     app: admission-webhook
 rules:
-- apiGroups: ["apps"]
-  resources: ["statefulsets"]
-  verbs: ["get"]
-- apiGroups: ["pingcap.com"]
-  resources: ["tidbclusters"]
-  verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["create", "get", "list", "watch", "update"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "create", "update", "delete"]
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "watch","update"]
```

Comment:
do we need write permissions (update/delete) on persistentvolumeclaims & pods?

Reply:
Currently we need to update pod and pvc during pd scale-in. We might need to delete pvc if we implement `create pod webhook` in future.

Reply:
I will remove `delete pod` permissions.